packages/orbit-components/config/deploymentUtils.mts

```diff
@@ -1,29 +0,0 @@
-import { Octokit } from "@octokit/rest";
```

Comment:
Is this library still used somewhere else?

If not, let's please clean that up, as well as possible env variables related to it.

Reply:
It is, I checked it. It is used to fetch the contributors to display on the documentation.

Reply:
Alright, thanks for checking! :)